### PR TITLE
Technical Docs Work group block and guide

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Create a Blog and Guide for contributing to technical docs

### CODE REVIEW

File: site/content/posts/2021-06-10-docs-wg.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+author_avatar: /img/contributors/orlin-vasilev.jpg
+categories: [kubernetes]
+tags: ['Contour Team', 'Abigail McCarthy', 'Docs' ,'Docs-WG', 'Jonas Rosland', 'Orlin Vasilev']
+date: 2021-06-10
+slug: docs-wg
+---
+
+
+
+# Join the Contour Tech Docs Working Group
```
Comment (jonasrosland) [inline site/content/posts/2021-06-10-docs-wg.md:14]: This should be the title on line 1, then this line can be removed

File: site/content/posts/2021-06-10-docs-wg.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The Contour team is excited to announce we are launching a technical documentation working group. This group will focus on improving the Contour website, documentation, and processes around content creation.
+
+# Because documentation makes a difference
+
+In 2017, GitHub conducted the Open Source Survey, which sampled 5,500 respondents. One of the key takeaways was that “documentation is highly valued, but often overlooked.” The survey found that outdated or confusing documentation was a major problem, affecting 93% of respondents. Yet 60% of contributors rarely or never contribute to documentation. We want this working group to be an opportunity to give back to the community by helping to improve the docs.
```
Comment (jonasrosland) [inline site/content/posts/2021-06-10-docs-wg.md:20]: Missing link to the survey: https://opensourcesurvey.org/2017/

File: site/content/posts/2021-06-10-docs-wg.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# The purpose and scope of the Working Group
+
+The purpose of the Working Group is to build up a self-sustaining community around documentation for Contour. We have an initial need to rework the existing documentation based on the recommendations outlined in the CNCF Tech Docs Review, and a continuous need for documentation being added/edited/removed for releases going forward.
```
Comment (jonasrosland) [inline site/content/posts/2021-06-10-docs-wg.md:24]: Missing link to the CNCF Tech Docs Review: https://github.com/cncf/techdocs/blob/main/assessments/0001-contour.md

File: site/content/posts/2021-06-10-docs-wg.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+This group is open to contributors of all levels, the only requirement is being interested in helping with docs!
+
+# HOW TO PARTICIPATE
+Join us for our kickoff meeting on June 17th at 1PM Eastern Standard/Daylight Time as part of the Contour Office Hours. During the kickoff meeting, we’ll have an information session about this working group, covering the following topics
```
Comment (jonasrosland) [inline site/content/posts/2021-06-10-docs-wg.md:28]: Missing link to time zone checker: https://www.timeanddate.com/worldclock/usa/boston

File: site/content/posts/2021-06-10-docs-wg.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+# Because documentation makes a difference
+
+In 2017, GitHub conducted the Open Source Survey, which sampled 5,500 respondents. One of the key takeaways was that “documentation is highly valued, but often overlooked.” The survey found that outdated or confusing documentation was a major problem, affecting 93% of respondents. Yet 60% of contributors rarely or never contribute to documentation. We want this working group to be an opportunity to give back to the community by helping to improve the docs.
+
+# The purpose and scope of the Working Group
+
+The purpose of the Working Group is to build up a self-sustaining community around documentation for Contour. We have an initial need to rework the existing documentation based on the recommendations outlined in the CNCF Tech Docs Review, and a continuous need for documentation being added/edited/removed for releases going forward.
+This group is open to contributors of all levels, the only requirement is being interested in helping with docs!
+
+# HOW TO PARTICIPATE
```
Comment (jonasrosland) [inline site/content/posts/2021-06-10-docs-wg.md:27]: Let's make this "How to participate"

File: site/contributing-docs.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+
```
Comment (jonasrosland) [inline site/contributing-docs.md:1]: This file should be located in https://github.com/projectcontour/contour/tree/main/site/content/resources

File: site/content/resources/contributing-docs.md [hunk 7]
```diff
@@ -0,0 +1 @@
+../../contributing-docs.md
```
Comment (jonasrosland) [inline site/content/resources/contributing-docs.md:1]: This isn't needed, just put the original file in the resources folder

File: site/content/posts/2021-06-10-docs-wg.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+- Working Group participation
+- The docs tool chain and how to set up your environment
+- How to preview the documentation locally before submitting a pull request
+- Anything else you’d like to know about helping with Contour documentation!
+
+We’ll be continuing to hold Working Group meetings and information sessions during the regular Contour Office Hours time every 1st and 3rd Thursday at 1PM after the kickoff meeting. 
+Get the invite to the Contour Office Hours by joining the mailing list.
+
+You can find the Guide [here](https://github.com/contour/contour/blob/docs-wg-guide/site/contributing-docs.md)
```
Comment (jonasrosland) [inline site/content/posts/2021-06-10-docs-wg.md:38]: This link is incorrect, should link to "/resources/contributing-docs" and it'll render correctly

File: site/contributing-docs.md (hunk 6 above)
Comment (jonasrosland) [reply site/contributing-docs.md:1]: Add the following front matter at the top of of this document:

```
---
title: Contributing Technical Documentation
layout: page
---
```

File: site/content/posts/2021-06-10-docs-wg.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+# HOW TO PARTICIPATE
+Join us for our kickoff meeting on June 17th at 1PM Eastern Standard/Daylight Time as part of the Contour Office Hours. During the kickoff meeting, we’ll have an information session about this working group, covering the following topics
+
+- Working Group participation
+- The docs tool chain and how to set up your environment
+- How to preview the documentation locally before submitting a pull request
+- Anything else you’d like to know about helping with Contour documentation!
+
+We’ll be continuing to hold Working Group meetings and information sessions during the regular Contour Office Hours time every 1st and 3rd Thursday at 1PM after the kickoff meeting. 
+Get the invite to the Contour Office Hours by joining the mailing list.
```
Comment (jonasrosland) [inline site/content/posts/2021-06-10-docs-wg.md:36]: Missing link to the mailing list: https://lists.cncf.io/g/cncf-contour-users/

**Review by jonasrosland — CHANGES_REQUESTED** [review]
Thanks Orlin! A few small changes requested.

File: site/content/posts/2021-06-10-docs-wg.md (hunk 1 above)
Comment (OrlinVasilev) [reply site/content/posts/2021-06-10-docs-wg.md:14]: fixed

File: site/content/posts/2021-06-10-docs-wg.md (hunk 3 above)
Comment (OrlinVasilev) [reply site/content/posts/2021-06-10-docs-wg.md:24]: fixed

**Comment by OrlinVasilev** [comment]
@jonasrosland all fixed

**Comment by jonasrosland** [comment]
LGTM, approved by me :)

**Review by jonasrosland — APPROVED** [review]

**Comment by OrlinVasilev** [comment]
@stevesloka and @skriss can you review that please! :)

**Review by skriss — APPROVED** [review]
LGTM, thanks @OrlinVasilev!

File: site/content/resources/contributing-docs.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+- Join the [Contour Office Hours Zoom](https://zoom.us/j/96698475744?pwd=KzVUd3BZSWI2bWIxTmhjZ2d5QVcxUT09) every 1st and 3rd Thursday at 1-2pm ET / 10am-11pm PT and introduce yourself
+- On the [Kubernetes Slack workspace](https://slack.k8s.io/), join the [#contour](https://kubernetes.slack.com/messages/contour) channel and introduce yourself
+
+When you introduce yourself, let us know:
+
+- Why you are interested in participating in the working group and what you hope to get out of your time contributing to the Contour docs
+- Your prior experience level with our docs toolchain (Git and GitHub, Markdown, Hugo, HTML) and where you need some help
+- Which operating system you’re using (for help with setting up your environment)
+
+We’ll work with you to decide which issue would be best suited for you to work on. We’ll then assign you to the open issue on GitHub for easy tracking.
```
Comment (a-mccarthy) [inline site/content/resources/contributing-docs.md:24]: Maybe update the last sentence to be more general?

```We can help you help find an open issue to work on or answer any questions you have about writing Contour docs. If you notice something about the docs you'd like to improve, please file an issue and bring it up with the working group. We'd love to hear your ideas.```

File: site/content/resources/contributing-docs.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+1. For new content for future versions, it should be created in `main`
+1. For edits to older versioned docs, first make the edits to that specific version (spelling errors, broken links etc) and then verify if those changes should also be incorporated in the `main` directory for the latest development docs
+
+### Create a Pull Request with your changes
+
+Please see the [CONTRIBUTING doc](https://github.com/projectcontour/contour/blob/main/CONTRIBUTING.md#contribution-workflow) in the section "Contribution workflow" for more detailed information on how to commit your changes and submit a pull request.
+
+### What if you can’t finish your work?
+
+By signing up to the working group and assigning issues, we assume you get pull requests for a specific issue started within a few days. However, if you are unable to finish within a few days, please submit a pull request with the content that you were able to create so far. We can then update the other team members on the progress through the GitHub issue, and someone else can continue the work. That way your work goes to use and you get credit :)
```
Comment (a-mccarthy) [inline site/content/resources/contributing-docs.md:50]: A suggestion for updating the ending to be more generic,

```
When you join the working group and get assigned an issue, we'd ask that you try to open a pull requests with your fixes within a few days. If you are unable to finish your assigned issue, please submit a pull request with the content that you were able to create and update the Github issue with the latest information of your progress. This way your work goes to use, you get credit for your a contributions, and we can work with other team members to continue making progress on the issue :)```

**Review by a-mccarthy — COMMENTED** [review]
@OrlinVasilev some suggestions on the contributing guide. Let me know your thoughts and if they make sense :)

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**